styles.css

```diff
@@ -360,3 +384,155 @@ textarea {
   background: #0d151e;
   border-radius: 10px;
 }
+
+.about_btn:hover {
+  border-radius: 10px 20px 10px 20px;
+}
+
+/* Media Queries */
+@media (min-width: 768px) {
+  header {
+    padding: 16px 143px;
+  }
+
+  div.hamburger {
+    display: none;
+  }
+
+  ul.nav_bar {
+    display: flex;
+    padding-left: 0;
+    list-style-type: none;
+  }
+
+  ul.nav_bar li {
+    padding: 0 12px;
+  }
+
+  ul.nav_bar .nav_links {
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  #headline {
+    background: url('/images/Header\ bg.svg'), #fff;
+
+    /* padding: 289px 260px; */
```

Comment:
- [ ] **[OPTIONAL]:** Please consider removing all pieces of commented code, this will make your codebase cleaner ending with better readability. If you're not using it, you most likely aren't going to need it! 🤓 🛠️